lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -281,6 +281,16 @@ def format_idempotency_key(options, suffix)
 
         options.merge(idempotency_key: "#{options[:idempotency_key]}-#{suffix}")
       end
+
+      def success_from(response, options)
+        if response['status'] == 'requires_action' && !options[:execute_threed]
```

Comment:
another thought related to question in PR description would be to add one more flag so that we can still standardize around `execute_threed` but explicitly request to require it, e.g. something like `if response['status'] == 'requires_action' && options[:require_threed_flag] && !options[:execute_threed]`...but that seems maybe overly complicated?

Reply:
To me, this does seem overly complicated. Spreedly currently does not use the `execute_threed` flag for Stripe Payment Intents implementation. This is something we'll need to add in core, right?

Reply:
n/m just saw your core update.

Reply:
yep correct, to take advantage of this, any integrations will need to pass in an `execute_threed` flag in their options params. This approach matches what we do in other AM adapters that allow for 3DS auth, like adyen: https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/adyen.rb#L530